project/npda/templates/dashboard.html

```diff
@@ -1,72 +1,75 @@
-{% extends 'base.html' %} 
-{% load static %} 
-{% load npda_tags %} 
-{% block content %} 
+{% extends 'base.html' %}
+{% load static %}
+{% load npda_tags %}
+{% block content %}
 {% url 'dashboard' as hx_get %}
 
-<section class="bg-rcpch_dark_blue font-montserrat">
-  <div class="py-8 px-4 mx-auto max-w-screen-xl text-center lg:py-16 lg:px-12">
-    <h1
-      class="mb-4 text-4xl font-extrabold font-arial tracking-tight leading-none text-gray-900 md:text-5xl lg:text-6xl text-white"
-    >
-      National Paediatric Diabetes Audit
-    </h1>
-    <p class="mb-8 text-lg font-normal lg:text-xl sm:px-16 xl:px-48 text-white">
-      We are now accepting data for the 2023-24 audit year.
-      <b
-        >{% text_for_data_submission can_upload_csv can_complete_questionnaire
-        %}</b
-      >
-      For more information check out our user guide.
-    </p>
-    <div
-      class="flex flex-col mb-8 lg:mb-16 space-y-4 sm:flex-row sm:justify-center sm:space-y-0 sm:space-x-4"
-    >
-      <a
-        href="http://0.0.0.0:8007/"
-        class="inline-flex justify-center items-center py-3 px-5 text-base font-medium text-center text-white rounded-lg bg-primary-700 hover:bg-primary-800 focus:ring-4 focus:ring-primary-300 focus:ring-primary-900"
-      >
-        Learn more<svg
-          class="ml-2 -mr-1 w-5 h-5"
-          fill="currentColor"
-          viewBox="0 0 20 20"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <path
-            fill-rule="evenodd"
-            d="M10.293 3.293a1 1 0 011.414 0l6 6a1 1 0 010 1.414l-6 6a1 1 0 01-1.414-1.414L14.586 11H3a1 1 0 110-2h11.586l-4.293-4.293a1 1 0 010-1.414z"
-            clip-rule="evenodd"
-          ></path>
-        </svg>
-      </a>
-    </div>
+<div class="container mx-auto p-4 font-montserrat text-rcpch_dark_blue">
+  <div class="my-6">
+    <h1 class="text-5xl text-center">PDU Dashboard for {{pdu_lead_organisation.name}} ({{pdu_object.pz_code}})</h1>
+    <p class="text-center text-gray-500 mt-2">Calculated at {{ kpi_calculations_object.calculation_datetime }}</p>
+    <p class="text-center">This data has not been validated by the NPDA team.</p>
   </div>
-</section>
-<div class="container-mx-auto my-20 mt-10 flex justify-center">
-  <div class="card bg-base-100 w-96 shadow-xl">
-    <figure>
-      <div id="organisation_cases_map" style="width: 100%; height: 700px"></div>
-    </figure>
-    <div class="card-body">
-      <h2 class="card-title">Patients</h2>
-        <p>Distribution of all patients at {{ session.pz_code }}</p>
+  <div class="grid grid-cols-1 md:grid-cols-3 gap-4">
+    <!-- PDU Information Card -->
+    {% include 'dashboard/pdu_card.html' %}
+
+
+    <!-- Audit Details Card -->
+    {% include 'dashboard/audit_details_card.html' %}
+
+    <!-- Total Eligible Patients Card -->
+    {% include 'dashboard/total_eligible_patients_card.html' %}
+  </div>
+
+  <!-- UNIT REPORT -->
```

Comment:
something happening here now with the rows since adding the deprivation map. I think the height generated by the map is being applied to all rows